recipe/meta.yaml

```diff
@@ -1,6 +1,5 @@
 {% set name = "osmnx" %}
 {% set version = "2.0.0" %}
-{% set python_min = "3.9" %}
```

Comment:
Where is `python_min` defined now? I don't see it, but it's still referenced throughout this yaml file.

Reply:
Please see @ocefpaf's comment:

> We only need to define this if the min python is higher then the EOL. Python 3.9 is conda-forge's min now.

Reply:
Got it. I see his comment now.